includes/class-blocks.php

```diff
@@ -84,7 +84,18 @@ public static function register_blocks() {
 		\register_block_type_from_metadata( ACTIVITYPUB_PLUGIN_DIR . '/build/extra-fields' );
 		\register_block_type_from_metadata( ACTIVITYPUB_PLUGIN_DIR . '/build/follow-me' );
 		\register_block_type_from_metadata( ACTIVITYPUB_PLUGIN_DIR . '/build/followers' );
-		\register_block_type_from_metadata( ACTIVITYPUB_PLUGIN_DIR . '/build/reactions' );
+		// Register reactions block, conditionally removing facepile style if avatars are disabled.
+		$reactions_args = array();
+		if ( ! \get_option( 'show_avatars', true ) ) {
+			$reactions_args['styles'] = array(
+				array(
+					'name'      => 'compact',
+					'label'     => __( 'Compact', 'activitypub' ),
+					'isDefault' => true,
+				),
+			);
```

Comment:
```suggestion
			$reactions_args['styles'] = array();
```

Removing all styles when we don't show avatars might be the best way forward?

With only Compact, the block still displays a Styles section but you can't change it, which feels awkward. With empty styles it defaults to compact without showing the styles section.

<img width="287" height="120" alt="Image" src="https://github.com/user-attachments/assets/a2899367-51f9-481d-9d87-5fa7d3f1831c" />